app/views/eligibility_interface/start/show.erb

```diff
@@ -1,44 +1,56 @@
 
-<p class="govuk-body">If you trained in England or Wales, or have not yet qualified, learn more about <%= govuk_link_to "other routes to QTS", "https://www.gov.uk/government/publications/apply-for-qualified-teacher-status-qts-if-you-teach-outside-the-uk/routes-to-qualified-teacher-status-qts-for-teachers-and-those-with-teaching-experience-outside-the-uk" %>.
+<p class="govuk-body"><strong>You must have:</strong></p>
 
-<p class="govuk-body">Use this service to:</p>
+<ul class="govuk-list govuk-list--bullet">
+  <li>an undergraduate degree equal to a UK bachelor’s degree</li>
+  <li>completed teacher training equal to a <%= govuk_link_to "level 6 qualification", "https://www.gov.uk/what-different-qualification-levels-mean/list-of-qualification-levels#:~:text=Level%206%20qualifications%20are%3A,graduate%20certificate" %></li>
+  <li>completed all mandatory induction processes to gain full teacher registration in the country where you qualified</li>
+  <li>been recognised as a professional teacher in the country where you qualified, and have no conditions or restrictions on your teaching practice</li>
+  <li>qualifications to show you can teach children aged anywhere between 5 and 16 years old</li>
+  <li>at least one school year (9 months) of experience employed as a teacher in any country, but must be from after you first became qualified to teach</li>
+</ul>
+
+<p class="govuk-body">Depending on the country that you trained to teach in, you may be required to have:</p>
 
 <ul class="govuk-list govuk-list--bullet">
-  <li>check if you’re eligible to apply for QTS</li>
-  <li>understand the types of evidence you’ll need to provide</li>
+  <li>qualifications to show you can teach children aged anywhere between 11 and 16 years old</li>
+  <li>a teaching specialism in one of the following subjects: maths, science, biology, chemistry, physics, French, German, Italian, Japanese, Latin, Mandarin, Russian, or Spanish</li>
 </ul>
 
-<p class="govuk-body">We’ll ask some questions to check your eligibility to apply for QTS in England. This should take about 5 minutes.</p>
+<p class="govuk-body">If you trained in England or Wales, or have not yet qualified, learn more about <%= govuk_link_to "other routes to getting QTS", "https://www.gov.uk/government/publications/apply-for-qualified-teacher-status-qts-if-you-teach-outside-the-uk/routes-to-qualified-teacher-status-qts-for-teachers-and-those-with-teaching-experience-outside-the-uk#routes-to-qts" %>.</p>
 
-<p class="govuk-body">There’s no charge to apply for QTS. You do not need to be in the UK to use this service.</p>
+<%= govuk_start_button(text: "Check you’re eligible", href: eligibility_interface_start_path, as_button: true) %>
```

Comment:
It looks like this button has moved to above the link to continue working on an existing application rather than below where it was before, is that correct?

Reply:
Yes the design document shows that this button goes above the link continue working on an existing application